R/auth.R

```diff
@@ -303,14 +303,8 @@ resendInvitation <- function(
   invisible(TRUE)
 }
 
-#' (Deprecated) List authorized users for an application
-#'
-#' @param appDir Directory containing application. Defaults to current working
-#'  directory.
-#' @export
+# Previously exported, but deprecated since 2015
 authorizedUsers <- function(appDir = getwd()) {
-  .Deprecated("showUsers")
```

Comment:
But we still use this internally so we're keeping it around? Is it not possible to shift everything to `showUsers`?

And if yes: did the `.Deprecated("showUsers")` get exposed to users in that internal use? Do we want to maintain that?

Reply:
Remarkably, yes, it is used internally in one place, and wrapped in `suppressWarnings()` presumably to squelch the deprecation warning. `showUsers` looked sufficiently different that I didn't want to swap it in, so instead of deleting this, I just made it internal.